tools/ping_enumerator.py

```diff
@@ -3,76 +3,141 @@
 """Scan serial ports for ping devices
     Symlinks to detected devices are created under /dev/serial/ping/
     This script needs root permission to create the symlinks
-    Exits with the number of ping devices detected
 """
+from __future__ import print_function
 import subprocess
-from brping import Ping1D
+import traceback
+import sys
+from brping import ping1d
+from brping import ping360
 
-def main():
-    """Main function
-        Enumerate new ping devices
+
+def detect_ping1d(device):
```

Comment:
refactor these two functions to be one function:

get_device_type(device) returning the device type from the `device_information` message.

Reply:
Ping1D doesn't reply that one yet... https://github.com/bluerobotics/ping-firmware-private/issues/15
Even if we implement that now, it would not detect the older ones.

Reply:
Add a function to 'legacy check' for old ping1d by requesting a message that only ping1d responds to.

Reply:
```
device = get_device_type()
if device == -1: # no response
  device = legacy_ping1d_probe()

if device == -1:
  #no device
```

Reply:
done